crates/tests/nightly_class_factory/tests/test.rs

```diff
@@ -28,7 +28,7 @@ impl IClassFactory_Impl for Factory {
         assert!(outer.is_none());
         let unknown: IInspectable = Object().into();
         // TODO: https://github.com/microsoft/windows-rs/issues/1441
-        unsafe { unknown.query(&*iid, object).ok() }
+        unsafe { unknown.query(&*iid, object as *mut _).ok() }
```

Comment:
It might be worth adding a `SAFETY` comment here, although I don't know that it'd be terribly informative. It looks like you're just calling an unsafe function, `query`, and following its requirements.

Reply:
I think that's a good idea, but we can take care of that when we document `query`'s safety needs (which I personally don't yet have a good grasp on).